doc/nrf/protocols/matter/end_product/test_event_triggers.rst

```diff
@@ -0,0 +1,274 @@
+.. _matter_test_event_triggers:
+
+Matter test event triggers
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Matter test event triggers are a set of software commands designed to initiate specific test scenarios.
+You can activate these commands using the Matter Controller, which serves as the interface for executing them.
+To activate a command, you need to enter the pre-defined activation codes that correspond to the desired test event.
+All test event triggers are based on the ``generaldiagnostics`` Matter cluster, and utilize the ``test-event-trigger`` command to function.
+This mechanism can be used during the certification process to simulate common and specific device scenarios, and during software verification events to validate the correct operation of Matter-compliant devices.
+
+When you provide the enable key using factory data, you can utilize the event trigger feature during the Matter Certification process.
+This is because, when done this way, you can turn off the event trigger functionality by disabling access to the ``generaldiagnostics`` cluster without altering the code that has already been certified.
+
+Once the certification process is complete, you can deactivate the test event trigger functionality by generating new factory data with a modified enable key value.
+This is done by setting the :kconfig:option:`CONFIG_CHIP_DEVICE_ENABLE_KEY` Kconfig option to a value consisting solely of zeros.
+
+For instance, to generate factory data with disabled event trigger functionality, set the :kconfig:option:`CONFIG_CHIP_DEVICE_ENABLE_KEY` Kconfig option to the value ``0x00000000000000000000000000000000``.
+After generating it, flash the :file:`factory_data.hex` file onto the device.
```

Comment:
I think we could probably move this information to be a separate section. I'd say on the same level and just before the section for the custom event triggers section, or as the very last section. Something like "Using in Matter certification".

Having its own section gives it visibility, and it could then be linked from the mention on Line 14.
```suggestion
When you provide the enable key using factory data, you can utilize the event trigger feature during the Matter certification process.
This is because, when done this way, you can turn off the event trigger functionality by disabling access to the ``generaldiagnostics`` cluster without altering the code that has already been certified.

Once the certification process is complete, you can deactivate the test event trigger functionality by generating new factory data with a modified enable key value.
This is done by setting the :kconfig:option:`CONFIG_CHIP_DEVICE_ENABLE_KEY` Kconfig option to a value consisting solely of zeros.

For instance, to generate factory data with disabled event trigger functionality, set the :kconfig:option:`CONFIG_CHIP_DEVICE_ENABLE_KEY` Kconfig option to the value ``0x00000000000000000000000000000000``.
After generating it, flash the :file:`factory_data.hex` file onto the device.
```